.github/workflows/deploy-web-to-dev.yml

```diff
@@ -0,0 +1,58 @@
+name: On Deployment Created
+on: 
+  deployment
+env:
+  PULL_NUMBER: ${{ github.event.deployment.payload.pr }}
+  MICRO_SERVICE: ${{ github.event.deployment.payload.microservice }}
+  DEPLOY_REF: ${{ github.event.deployment.ref }}
+  DEPLOYMENT_ID: ${{ github.event.deployment.id }}
+  ENVIRONMENT: ${{ github.event.deployment.environment }}
+  INFRA_NAME: devhub-app-web
+  SUFFIX: "-${{ github.event.deployment.payload.pr }}"
+jobs:
+  build:
+    if: ${{ github.event.deployment.payload.microservice == 'web' }}
+    name: Build and Deploy Gatsby
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      
+      - name: Change Deploy Status to Pending
+        uses: actions/github-script@0.3.0
+        with:
+          github-token: ${{github.token}}
+          script: |
+            github.repos.createDeploymentStatus({
+              deployment_id: context.payload.deployment.id,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'pending'
+            });
+      - name: Cluster Login
+        uses: redhat-developer/openshift-actions@v1.1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT_SERVER_URL}}
+          parameters: '{"apitoken": "${{ secrets.SA_PASSWORD }}"}'
```

Comment:
that's the SA `github-cicd` created in cicd.yaml?

Reply:
yep! I'm also thinking of creating a secondary service account to serve test and prod. all it will be allowed to do is create/import images and run deployments